AMDS_sgm.pro

```diff
@@ -1,7 +1,14 @@
 
-PATH_TO_SOURCE = /home/liux0/beamline/programming/AcquamanDataServer/source
-PATH_TO_ACQUAMAN = /home/liux0/beamline/programming/acquaman/source
-PATH_TO_MPLOT = /home/liux0/beamline/programming/MPlot/src
+HOME_FOLDER = $$system(echo $HOME)
+
+USERNAME = $$system(whoami)
+contains(USERNAME, liux0) {
+	HOME_FOLDER = "/home/liux0"
+}
+
```

Comment:
Is there a reason why (echo $HOME) doesn't work for your username, and is there likely to be similar issues on any of the other SL machines (which will be the deployment environment)?